src/routes.js

```diff
@@ -20,6 +22,11 @@ const routes = (
     <Route path="CanWeHelpYou" component={CanWeHelpYou}/>
     <Route path="OurVoice" component={OurVoice}/>
     <Route path="WantToLearn" component={WantToLearn}/>
+    <Route path="OurVoice" component={OurVoice} >
+      {/* I can't figure out why this route won't work as an IndexRoute */}
```

Comment:
maybe the space in the markdown filenames has something to do with it ?
wild guess here